instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/HttpRouteHolder.java

```diff
@@ -151,10 +156,38 @@ private boolean isBetterRoute(String name) {
    */
   @Nullable
   static String getRoute(Context context) {
-    HttpRouteHolder httpRouteHolder = context.get(CONTEXT_KEY);
+    HttpRouteHolder httpRouteHolder = context.get(KEY);
     return httpRouteHolder == null ? null : httpRouteHolder.route;
   }
 
+  /**
+   * This method is internal and is hence not for public use. Its API is unstable and can change at
+   * any time.
+   */
```

Comment:
I don't think these disclaimers are needed (guessing they were originally public before changing access to reflection?)

Reply:
wait, how is the MethodHandle able to access this? 🤔

Reply:
Removed the disclaimer comments.